esmlab/utils/time.py

```diff
@@ -52,109 +39,167 @@ def time_attrs(self):
         attrs = self.time.attrs
         encoding = self.time.encoding
 
-        if "units" in attrs:
-            units = attrs["units"]
-        elif "units" in encoding:
-            units = encoding["units"]
+        if 'units' in attrs:
+            units = attrs['units']
+        elif 'units' in encoding:
+            units = encoding['units']
         else:
             units = None
 
-        if "calendar" in attrs:
-            calendar = attrs["calendar"]
-        elif "calendar" in encoding:
-            calendar = encoding["calendar"]
+        if 'calendar' in attrs:
+            calendar = attrs['calendar']
+        elif 'calendar' in encoding:
+            calendar = encoding['calendar']
         else:
             calendar = DEFAULT_CALENDAR
 
-        if "bounds" in attrs:
-            bounds = attrs["bounds"]
-        elif "bounds" in encoding:
-            bounds = encoding["bounds"]
+        if 'bounds' in attrs:
+            bounds = attrs['bounds']
+        elif 'bounds' in encoding:
+            bounds = encoding['bounds']
         else:
             bounds = None
 
-        return {"units": units, "calendar": calendar, "bounds": bounds}
+        key_attrs = {'units': units, 'calendar': calendar, 'bounds': bounds}
+        other_attrs = {k: v for k, v in attrs.items() if k not in key_attrs}
+        return dict(**key_attrs, **other_attrs)
```

Comment:
Apparently, Python 2 is not happy about this syntax. Do you know why?

Reply:
no. Can you fix and push to my branch?

Reply:
Sounds good. I am going to take a stab at coming up with a fix

Reply:
I think

```
new_attrs = {}
new_attrs.update(key_attrs)
new_attrs.update(attrs)
return(new_attrs)
```

does what you want, and is compatible with both py2 and py3